uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/sub-controllers/leave-request-ctrls.spec.js

```diff
@@ -1288,6 +1288,137 @@
         });
       });
 
+      describe('getStatuses', function () {
+        var status = {};
+
+        beforeEach(function () {
+          var collectionName = 'hrleaveandabsences_leave_request_status';
+          var allStatuses = optionGroupMock.getCollection(collectionName);
+
+          status = _.indexBy(allStatuses, 'name');
+
+          initTestController({
+            contactId: CRM.vars.leaveAndAbsences.contactId,
+            userRole: 'admin'
+          });
+
+          $ctrl._init();
+          $ctrl.request = {
+            status_id: null
+          };
+          $ctrl.requestStatuses = status;
+        });
+
+        it('defines a getStatuses method', function () {
+          expect(typeof $ctrl.getStatuses).toBe('function');
```

Comment:
I dont think we need this test to be honest. Just below you are calling `$ctrl.getStatuses()`, and if it is not a function, we will get an error anyway.